### ISSUE TITLE
REST-API returns always true

### ISSUE THREAD

**Issue by kenci** [pimatic/pimatic#798]
When requesting status of a device over the REST-API you got something like this
```
{
  "device": {
    "id": "smartphone-kenan",
    "name": "Kenan",
    "template": "presence",
    "attributes": [
      {
        "description": "Presence of the human/device",
        "type": "boolean",
        "labels": [
          "present",
          "absent"
        ],
        "label": "Presence",
        "discrete": true,
        "name": "presence",
        "value": true,
        "history": [
          {
            "t": 1453960504445,
            "v": true
          },
```

But the problem is when i request a device that doesn't exist. It returns `{
  "success": true
}` as result.

**Comment by SLiX69** [comment]
```{ "success": true }``` is returned when your connection to the host was successful. If you look for a variable with a wrong id the API return ```{ "variable": null }, { "success": true }``` Would love to have that for everything. 

But a possible solution would be to look if the object ```"device": { }``` exists or is empty (don't know your possibilities (in NodeRed))

**Comment by sweetpi** [comment]
Indeed I think it would be better to return an error code 404. I will have a look into it.

**Comment by sweetpi** [comment]
Fixed in v0.9, will be released soon
